include/GafferImage/ImageReader.h

```diff
@@ -110,10 +110,18 @@ class ImageReader : public ImageNode
 		Gaffer::StringPlug *colorSpacePlug();
 		const Gaffer::StringPlug *colorSpacePlug() const;
 
+		Gaffer::StringPlug *colorSpaceMetadataNamePlug();
```

Comment:
Yeah, looking back through my email, looks like that was it. We only support Data in `ImageReader`, `ImageWriter`, `ImageMetadata` and `ImagePrimitive::blindData()` though, so how about I chuck together a quick PR to add a AtomicCompoundDataPlug and use it for `ImagePlug::metadataPlug()`?